From 97682a793abaec7e05ea99a9b59a901319562a30 Mon Sep 17 00:00:00 2001
From: He Zhe <zhe.he@windriver.com>
Date: Tue, 20 Jan 2015 14:59:06 +0800
Subject: [PATCH 10/10] settime: Replace int with ssize_t

Because the write pointer type is different from the function define,
there is a warning information caused by a35378a as follows:
notify_settime.c:99:2: warning: initialization from incompatible pointer type
  .write = notify_settime_signal_proc_write,
    ^
For fixing it, Replace int with ssize_t.

Signed-off-by: Jianchuan Wang <jianchuan.wang@windriver.com>
Signed-off-by: He Zhe <zhe.he@windriver.com>
---
 kernel/notify_settime.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/kernel/notify_settime.c b/kernel/notify_settime.c
index 56e8579..1b65894 100644
--- a/kernel/notify_settime.c
+++ b/kernel/notify_settime.c
@@ -73,7 +73,7 @@ static int notify_settime_signal_proc_open(struct inode *inode, struct file *fil
 	return single_open(file, notify_settime_signal_show, PDE_DATA(inode));
 }
 
-static int notify_settime_signal_proc_write(struct file *file,
+static ssize_t notify_settime_signal_proc_write(struct file *file,
 					const char __user *buffer,
 				       size_t count, loff_t *pos)
 {
-- 
1.7.5.4

